project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -0,0 +1,157 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.maps.model.LatLng;
+import com.google.maps.model.PlaceDetails;
+import com.google.maps.model.PlaceType;
+import com.google.maps.model.PlacesSearchResponse;
+import com.google.maps.model.PriceLevel;
+import com.google.maps.GeoApiContext;
+import com.google.maps.TextSearchRequest;
+import com.google.maps.PlaceDetailsRequest;
+import com.google.maps.errors.ApiException;
+import com.google.maps.model.PlacesSearchResult;
+import com.google.maps.PlacesApi;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.appengine.repackaged.com.google.common.collect.ImmutableList;
+
+public class PlacesFetcher {
+
+    /** Places will be fetched within a ceratain radius from this location. */
```

Comment:
1. Yeah I added the javaDoc because of the linter errors. I will try to remove the error.
2. I added them as class fields to avoid passing them to the fetch() and from fetch() to getPlacesSearchResults(), but I understand now that it is better to pass them to each function and keep the ability to instantiate a single instance of this class.
I changed the values to be constants for M0 as suggested.